pyproject.toml

```diff
@@ -33,7 +33,7 @@ maintainers = [
 license = { file = "LICENSE" }
 dependencies = [
     "nomad-lab>=1.3.16.dev100",
-    "nomad-simulations@git+https://github.com/nomad-coe/nomad-simulations.git@develop",
+    "nomad-simulations",
```

Comment:
I think we should for the moment use develop

Reply:
thank you for catching this. this is my local setup, didn't mean to include it here